src/background/plugins.js

```diff
@@ -81,10 +81,33 @@ export default {
     displayName: 'DuckDuckGo search',
     match: /\s*(.*?)/,
     keys: [ 'd', 'duckduckgo' ],
-    item: {
-      favicon: 'https://duckduckgo.com/favicon.ico',
-      title: 'Search DuckDuckGo for $1',
-      url: 'https://duckduckgo.com?q=$1',
+    async item(query) {
+      const results = [
+        {
+          favicon: 'https://duckduckgo.com/favicon.ico',
+          title: 'Search DuckDuckGo for $1',
+          url: 'https://duckduckgo.com?q=$1',
+        }
+      ]
+      if (query.length > 3) {
+        const search = await fetch(`https://api.duckduckgo.com/?q=${query}&format=json`).then(response => response.json())
+        if (search.Results[0]) results.push({
```

Comment:
Ok, I'll add all results, I put just the first one because *DDG* always return just one but is an array, and might be some more results in the future.